services/openshift-service-impl/src/test/java/org/kontinuity/catapult/service/openshift/impl/OpenShiftServiceTestBase.java

```diff
@@ -40,10 +63,20 @@ public void deleteCreatedProject() {
     
     @Test
     public void createProject() {
```

Comment:
Let's leave `createProject` as-is; its job was to test the creation of a project.

Probably we want to add a *new* test for `configurePipeline`, which will also create a project.  But this way we'll know if project creation is working independently of the pipeline creation stuff.